qemu/qemu.go

```diff
@@ -191,6 +191,13 @@ type NetDevice struct {
 
 	// Script is the tap interface configuration script.
 	Script string
+
+	// FDs represents the list of already existing file descriptors to be used.
+	// This is mostly useful for mq support.
+	FDs []int
```

Comment:
We need to figure out how to fold in fd's from multiple devices into the qemu invocation.. @markdryan had observed some fd number movement as fd's moved from parent to child.